packages/core/src/browser/menu/browser-menu-plugin.ts

```diff
@@ -120,7 +120,7 @@ class DynamicMenuBarWidget extends MenuBarWidget {
 /**
  * A menu widget that would recompute its items on update
  */
-class DynamicMenuWidget extends MenuWidget {
+export class DynamicMenuWidget extends MenuWidget {
```

Comment:
not needed when using phosphor menu directly.

Reply:
fixed